dev-site-config/site.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  filterSideMenu: true,
```

Comment:
Should we wait to update this config for when terra-dev-site is released?

Reply:
I think this should be fine, I don't see it having any impact if the config doesn't do anything in terra-dev-site yet.